.storybook/main.js

```diff
@@ -3,10 +3,6 @@ module.exports = {
       "../src/**/*.stories.mdx",
       "../src/**/*.stories.@(js|jsx|ts|tsx)"
     ],
-    "addons": [
```

Comment:
Why are these being removed? Just double checking.

Reply:
I don't think we ever needed them. So would be better simply removing them